docs/reference/indices/diskusage.asciidoc

```diff
@@ -0,0 +1,160 @@
+[[indices-disk-usage]]
+=== Analyze index disk usage API
+++++
+<titleabbrev>Analyze index disk usage</titleabbrev>
+++++
+
+experimental[]
+
+Analyzes the disk usage of each field of an index. This API
+might not support indices created in previous {es} versions.
```

Comment:
Also maybe there should be a note that this API only works well on large indices, since it ignores the disk footprint of metadata (field info, and more generally all the things that Lucene reads upon opening the index).